tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -2721,13 +2726,27 @@ tensorflow::Status ConvertGraphDefToEngine(
         StrAppend(&dim_str, " ]");
         VLOG(1) << dim_str;
       }
+
+#if NV_TENSORRT_MAJOR == 3
+      nvinfer1::DimsCHW input_dim;
```

Comment:
Would you please put these inside ValidateInputeProperties()?

Reply:
A second thought, we should actually put it in InputEdgeValidator. As this check only applies to live-in tensors